scripts/validate_container.sh

```diff
@@ -245,6 +216,38 @@ do
     echo "============================="
 done
 
+echo "============================="
```

Comment:
I can't tell - did anything change in this snippet, or was it just moved?

Reply:
Never mind, I compared using an external diff tool. I see that 🏳️ , ✅ , and  ❌ were added and no other changes were made (other than moving it). Just curious what the reason for moving it was.

Reply:
It was moved and I added output to make it also show up in the checklist in the step summary.

Reply:
Not much - I added the notebook validation below (separate PR), so I moved the Python part down as well.